apps/settings/js/modules/date_time.js

```diff
@@ -193,9 +193,7 @@ define(function(require) {
   DateTime.prototype.handleEvent = function(evt) {
     switch (evt.type) {
       case 'moztimechange':
-        navigator.mozL10n.ready((function _updateTime() {
-          this._autoUpdateDateTime();
-        }).bind(this));
+        this._autoUpdateDateTime();
```

Comment:
I'm curious why you removed `navigator.mozL10n.ready` and `navigator.mozL10n.once` everywhere. :-/